src/scss/06_components/forms/_collapsible.scss

```diff
@@ -0,0 +1,82 @@
+$collapsible-height: 56px;
+
+.collapsible-wrapper {
+  position: relative;
+}
+
+.collapsible__element {
+  border: 2px solid #808080;
+}
+
+.collapsible-container {
+  align-items: center;
+  background: $color-white;
+  display: flex;
+  height: $collapsible-height;
+  margin-bottom: $spacing;
+  padding: $spacing;
+}
+
+.collapsible__title {
+  width: calc(100% - $spacing-and-half);
+}
+
+.collapsible__control {
+  align-items: center;
+  background: $color-white;
+  display: flex;
+  font-size: 18px;
+  height: 56px;
+  padding: 12px 16px;
```

Comment:
Use variables here. The 12px should be explained in the comment if there is a reason for us to use padding that is not defined as a constraint variable. Check the `_constraint_variables.scss`. The 16px is `$spacing`.